demo/api/src/open-telemetry/metrics.ts

```diff
@@ -39,19 +39,3 @@ export function getOrCreateHistogram(name: string, options: MetricOptions = {}):
 export function getOrCreateCounter(name: string, options: MetricOptions = {}): Counter {
     return getOrCreate(name, options, MetricType.Counter) as Counter;
 }
-
-export function getOrCreateUpDownCounter(name: string, options: MetricOptions = {}): UpDownCounter {
-    return getOrCreate(name, options, MetricType.UpDownCounter) as UpDownCounter;
-}
-
-export function getOrCreateObservableGauge(name: string, options: MetricOptions = {}): ObservableGauge {
-    return getOrCreate(name, options, MetricType.ObservableGauge) as ObservableGauge;
-}
-
-export function getOrCreateObservableCounter(name: string, options: MetricOptions = {}): ObservableCounter {
-    return getOrCreate(name, options, MetricType.ObservableCounter) as ObservableCounter;
-}
-
-export function getOrCreateObservableUpDownCounter(name: string, options: MetricOptions = {}): ObservableUpDownCounter {
-    return getOrCreate(name, options, MetricType.ObservableUpDownCounter) as ObservableUpDownCounter;
-}
```

Comment:
These were recently introduced by @nsams, so maybe we shouldn't remove them?

Reply:
ok, but @nsams is not using those function. In my opinion we should remove that dead code.

Reply:
We will add more metrics in future - that might need those functions. It's a implementation for all available metric types. Please keep it (this is demo). Feel free to comment them out.

Reply:
added out commented code back. My heart bleeds 🩸 a little bit.